common/src/services/auth.service.ts

```diff
@@ -115,16 +116,19 @@ export class AuthService implements AuthServiceAbstraction {
     return result;
   }
 
-  async logInTwoFactor(twoFactor: TokenRequestTwoFactor): Promise<AuthResult> {
+  async logInTwoFactor(
+    twoFactor: TokenRequestTwoFactor,
+    captchaResponse: string
```

Comment:
I _think_ this can be removed now? Per other feedback, the captchaResponse is persisted in the logInStrategy, so we don't need to return it from `handleCaptchaRequired`. But that also means that we don't need to pass it to `authService.logInTwoFactor` or up the chain to the strategy. So I think you can revert this change, plus the changes in the logInStrategy base + sub class.

Reply:
In the `cli` flow that is correct. However, because captcha can be triggered **on** the `two-factor` component on the other clients now, we must pass in the `captchaResponse` (the response retrieved locally from the captcha validation) which is used on the server to approve the captcha required and continue to the 2fa validation and allow login to complete.

Reply:
Ah that's a good point